sphinx/application/common/wrappers/wrapper-message/src/main/java/chat/sphinx/wrapper_message/MessageType.kt

```diff
@@ -31,6 +31,10 @@ inline fun MessageType.isDirectPayment(): Boolean =
 inline fun MessageType.isAttachment(): Boolean =
     this is MessageType.Attachment
 
+@Suppress("NOTHING_TO_INLINE")
```

Comment:
can be removed. `MessageType.isBoost()` is already an extension function for type 29 messages.

Reply:
MessageType.isPodcastBoost removed